jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -42,6 +42,7 @@ function prepareEnv() {
     unset KIE_SERVER_SYNC_DEPLOY
     unset MYSQL_ENABLED_TLS_PROTOCOLS
     unset PROMETHEUS_SERVER_EXT_DISABLED
+    unset OPTAPLANNER_SERVER_EXT_THREAD_POOL_QUEUE_SIZE
```

Comment:
Document this new env on the jboss-kie-server image.yaml file for rhdm and rhpam.
i.e. https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/master/kieserver/image.yaml#L28

Reply:
PRs sent